survey.py

```diff
@@ -12,16 +12,21 @@ def get_args():
         print("Unknown Error! See full details below")
         raise
 
-
 def main():
     filename = get_args() #Get filename for sys arg
```

Comment:
Hey - just a question for my own understanding, how does this function get the filename of the input csv? Is it just taken from the command line?